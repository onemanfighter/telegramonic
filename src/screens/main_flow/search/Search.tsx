import { Box, Text } from '@chakra-ui/react';
import { SearchProps } from './types';

const Search = (props: SearchProps) => {
  return (
    <Box>
      <Text>Search</Text>
    </Box>
  );
};

export default Search;
