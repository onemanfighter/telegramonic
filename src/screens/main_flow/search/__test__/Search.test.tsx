import { render } from '@testing-library/react';
import Search from '../Search';

describe('Search screen', () => {
  it('should render Search screen', () => {
    const { container } = render(<Search />);

    expect(container).toMatchSnapshot();
  });
});
