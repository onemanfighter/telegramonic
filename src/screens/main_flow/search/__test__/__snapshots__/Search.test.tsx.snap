// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search screen should render Search screen 1`] = `
<div>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-0"
    >
      Search
    </p>
  </div>
</div>
`;
